src/app/models/propTypes/inline/index.js

```diff
@@ -1,13 +1,14 @@
-import { string, arrayOf, oneOf, shape } from 'prop-types';
+import { string } from 'prop-types';
+import { blockOfTypesAndModel, blocksWithTypes } from '../general';
 import fragmentBlockPropTypes from '../fragment';
 
-const inlineBlockPropTypes = shape({
-  type: 'inline',
-  model: shape({
-    text: string.isRequired,
-    blocks: arrayOf(oneOf([fragmentBlockPropTypes])),
-    language: string.isRequired,
-  }),
-});
+export const inlineModelPropTypes = {
```

Comment:
Should this be a shape still?

Reply:
In this context it needs to be a propType object not a `shape`, see for comparison https://github.com/bbc/simorgh/blob/latest/src/app/models/propTypes/inlineLink/index.js#L5

Will be utilised here: https://github.com/bbc/simorgh/pull/2791/files#diff-c9811bf222651dd1e0e5f119bdd0a118R21